README.md

```diff
@@ -711,7 +711,152 @@ config.customActionDelegate = YourCustomActionDelegate()
+
+![Linking](images/mobile-inbox.png)
+
+To configure an in-app message to use the inbox, marketers should:
+
+- Enable the **Show in Inbox** toggle in the in-app message's template
+- Choose whether or not to **Deliver Silently to Inbox** (so that the message does not display prominently when the device receives it, but can be viewed later by the user)
+
+#### Mobile Inbox implementation guide
+
+When implementing an inbox, you'll work primarily with the `IterableInboxNavigationViewController` class, a subclass of `UINavigationController`. This class automatically instantiates `IterableInboxViewController` as its root view controller.
+
+To avoid the use of a navigation controller, instantiate an `IterableInboxViewController` object. In this case, when reading the below documentation, substitute `IterableInboxViewController` for `IterableInboxNavigationViewController`.
+
+##### Show the inbox as a tab
+
+Many iOS apps place a message inbox in a `UITabBarController` tab. When using `IterableInboxNavigationViewController` as a tab's view controller (whether in a storyboard or in code), the tab automatically shows a badge count that represents the user's total unread in-app message count.
+
+##### Display mobile inbox as a result of an user action
+
+It's also possible to show the inbox when a user taps a button (or completes some other action). To do this, respond to the user's action by instantiating the `IterableInboxNavigationViewController` and displaying it. For example:
+
+```swift
+// Add this in your button tap handler
+let inboxViewController = IterableInboxNavigationViewController()
+present(inboxViewController, animated: true)
+```
+
+#### Mobile Inbox customization options
+
+The SDK provides various ways to customize the inbox's interface to match your brand's styles. For simpler customization needs, specify a custom `cellNibName` (and corresponding XIB file) on the `IterableInboxNavigationController`. For more advanced scenarios, define a class that inherits from `IterableInboxViewController` or `IterableInboxNavigationController` and override existing methods as necessary.
+
+##### inboxMode
+
+`IterableInboxViewController` can display inbox messages in two ways:
+
+- `popup` - (default): Displays the inbox message modally.
+- `nav` - Pushes the inbox message onto the navigation stack.
+
+It's generally best to use `nav` mode, since it makes sense in more app navigation paradigms.
+
+##### cellNibName
+
+To completely change the look of an inbox, create a custom `UITableViewCell` XIB file. Set the custom class of this XIB to `IterableInboxCell`, connect the outlets, and set the `cellNibName` property on `IterableInboxNavigationViewController` to the name of the newly created XIB file.
+
+##### navTitle
+
+To change the title of the inbox, set the `navTitle` property on `IterableInboxNavigationViewController`. Choose a simple title such as "Inbox".
+
+#### Mobile inbox events and the events lifecycle
```

Comment:
These events will be listed in the support docs. I wonder if they're necessary here? Maybe we keep them for now but remove them (or shorten them up significantly) once the updates to the support docs have been published?

Reply:
Yes. I think we should definitely revisit the Github readme once the support docs are published. Not only for inbox but for other items as well.